src/_posts/2020-09-17-cassandra-usage-report-2020.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Cassandra Usage Report 2020"
+date:   2020-09-17 09:00:00 -0700
```

Comment:
also in the previous fix https://github.com/loganloganlogan/cassandra-website/commit/ed73115251f2f4ffd83cb3edfb0008e6b0764f8a#diff-bedb8fd638d536c9b7ebfd8dabd55b4b I had to adjust the timezone… 

the best is to remove the timezone offset `-0700` altogether. it is [optional](https://jekyllrb.com/docs/front-matter/#predefined-variables-for-posts), and only useful when having to order blog posts from the same day.